src/repositories/LeaderboardRepository/LeaderboardRepository.ts

```diff
@@ -1,19 +1,28 @@
+import { Leaderboard, Prisma, PrismaClient } from "@prisma/client";
 import getEnvVariable from "../../utils/getEnvVariable";
-import NotionClient from "../helpers/NotionClient";
-import NotionElementHelper from "../helpers/NotionElementHelper";
-import {
-  LeaderboardPageResponseProperties,
-  LeaderboardPageRequestProperties,
-  PlayerStats,
-  PlayerStatsUpdates,
-} from "./types";
+import { LeaderboardWithBallChaser, PlayerStats, UpdatePlayerStatsInput } from "./types";
 
 export class LeaderboardRepository {
-  #Client: NotionClient<LeaderboardPageResponseProperties, LeaderboardPageRequestProperties>;
+  #Leaderboard: Prisma.LeaderboardDelegate<Prisma.RejectOnNotFound | Prisma.RejectPerOperation | undefined>;
+  #seasonSemester: string;
+  #seasonYear: string;
 
   constructor() {
-    const databaseId = getEnvVariable("notion_leaderboard_id");
-    this.#Client = new NotionClient(databaseId);
+    this.#Leaderboard = new PrismaClient().leaderboard;
+    this.#seasonSemester = getEnvVariable("season_semester");
```

Comment:
The day seasons stop/start vary so there's no way we can reliably programmatically set the season. Right now they are env variables which when changed will restart Norm and re-initialize the leaderboard repository to use the new season keys. Because Norm restarts, the leaderboard channel then gets refreshed with an empty leaderboard on startup.

This may change at some point in the future to maybe adding an admin command to set the new season, but I figured this was good enough for now